cfgov/jinja2/v1/_includes/ask/ask-search-bar.html

```diff
@@ -4,15 +4,16 @@
 {% set default_classes = 'block__bg-branded block__flush-top block__flush-bottom' %}
 {% set default_position = 'full' %}
 
-{% macro render( position=default_position, label=default_label, additional_classes=default_classes ) %}
+{% macro render( position=default_position, label=default_label, additional_classes=default_classes, q='' ) %}
 
 {% set search_opts = {
     'label': label,
     'position': position,
     'action': '/ask-cfpb/search',
     'additional_classes': additional_classes,
-    'autocomplete': 'true'
+    'autocomplete': 'true',
+    'q': q
```

Comment:
Trivial: Add a comma here so that this line doesn't have to get modified next time we have to add something else to this object :)

Reply:
The comma of hope.